test.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash -ex
+
+set -a
+
+: ${RUBY_VERSION=2.2}
+sed "s/\${RUBY_VERSION}/$RUBY_VERSION/" Dockerfile > Dockerfile.$RUBY_VERSION
```

Comment:
Yeah, I was really disappointed to discover that I needed 17.06, i.e. the broken release, to be able to do variable substitution in `FROM`. I saw the doc about the feature, wrote my Dockerfile using it, then discovered it didn't work in 17.03. :(